### PR TITLE
Expert Tuning

### PR DESCRIPTION
**Pull request by MuteTiefling**
## Create
Further cleanup of unused brass and other components
Extendo Grip
image

Rotation Speed Controller
image

Train Controls
image

Train Doors and Trapdoors
image
image

Contraption Barge
image

Flywheel
image

Nixie Tubes
image

Sequenced Gearshift
image

Adjustable Chain Gearshift
image

Electron Tubes replaced by Logistics Cores in any remaining recipes

## Immersive Engineering
Moving around a lot of crafting components

Vacuum Tube
image

Light Bulb
image

Ammo
Makes revolvers available early and buffs them considerably , making them a powerful alternative to spells or other ranged attacks. Effectively, everything is renewable to avoid spending precious metals on ammo.

Empty Shells
image

Empty Casings
image

Casull Cartridges
image

Buckshot
image

Every other ammunition is some modification on top of these two, keyed in by some form of essence. Like Homing and Armor Piercing below
image
image

Gun Crafting:
image
image
image
image

The parts may be enchanted too with a few pre-defined enchant levels. Nothing crazy, and certainly not as good as can be traded for with villagers, but with less RNG. 
image

Gun Upgrades
image
image
image

Other Cleanup
Item/Fluid Routers and batchers
image
image
image

Fluid Pump
image

External Heater
image

Cloche
image

Railgun
Has received a massive boost in power. 
image
Railgun Heatsinks
image
Scope
image

Turrets
image
image

Renamed and altered Electronic Components
image

Disable unused Metal Press Molds. Remaining molds crafted through enchantment:
image
image

Cheaper(?) recipes for Create Casings
image

## Pneumaticcraft
Adjustment to Module Expansion Crystal
image

Sentry Turret
image

Minigun
image

Ammo
All ammo follows this general template with the conjuration essence representing the ability to regenerate ammo with air.. The total ammo in each box has been buffed as well. 
image

Villager renamed and trades rebalanced to contain some PNC and AE2 elements. AE2's villager is otherwise inaccessible since the Charger is disabled. 
image

Upgrade Matrix
image

Upgrades moved to more magical methods of crafting with upgrades meant to go in machines (speed, volume, security, etc) generally getting automation friendly recipes such as these
image

Armor focused upgrades get more of a special treatment. Jet Boots and Jumping are both reduced to 3 tiers, 3, 4, 5 and  2, 3, 4 respectively. They're properly renamed to just 1, 2, and 3 however. 

Jet Boots
image
image
image

Jump Boost
image
image
image

The armor itself may be made from a variety of armors. This keeps the NBT of the original armor, so enchants and apotheosis affixes are preserved
image

Vacuum Pump
image

## Modular Routers
Most modules now require power. Also buffs power and fluid transfer rates.

As a result of the power requirements, I feel they can be fairly cheap to craft.
Routers
image

Blank cards
image
image

Stack
image

Speed
image

Fluid Transfer
image

Energy capacity
image

Vacuum
image

MK1 are normally required for MK2 crafts. However, the NBT on these caused some severe frustration with autocrafting in e6e, so I'm just de-coupling them entirely.

Fluid Module
MK1
image
MK2
image

Senders
Mk1
image

Mk2
image

Mk3
image

Pullers
Mk1
image

Mk2
image

Distributors
image

Energy Output (sends to adjacent block only)
image

Energy Distributor (sends to up to 8 remote blocks wirelessly)
image

Player Module
This should be very useful for mining trips as it can be used to automatically pull stuff back from the player inventory. Also great to combine with the IE revolver changes to keep speedloaders re-filled.
image

## Ars Nouveau
Thread
image

Craftable Elytra to enable a few crafts that use them.
image
Like glide threads
image

## Powah
Player Transmitters 
image

Upgrades are simple shapeless crafts
image

Binding Card
image
Dimensional Binding Card
image

## Mining Gadgets
To go along with the other tools available at this point, mining gadgets are completely re-worked. Power costs are increased substantially, and storage on the device is minimal, so a Player Transmitter is going to be key to keeping it running.

The gadgets themselves all follow the same base recipe. Just the nuggets change to set the visual type. 
image

Upgrade Cards
image

And all upgrades are essentially stamping a glyph onto an upgrade card.
The cost to craft is essentially very cheap, but that's made up for by the cost to operate. 
image
image
image

3x3 upgrade is disabled. The thinking here is that this tool gives extreme range compared to the other aoe options. And while Ars Spells can do AoE at range too, those spells tend to be costly and you have to trade something for it. 

Battery upgrades are disabled too, further cementing the need for wireless power. 

Modification Table
image

### CODE REVIEW

**Comment by MuteTiefling** [comment]
Primarily focused on shuffling around utility items to make them available at appropriate times. 

This also contains the following fixes for normal, which I'll update once this is approved/merged

Improvements
-   AE2 Growth Accelerators now work on Blue Skies Sunstone and Moonstone and BYG crystals, in addition to certus and amethyst

Fixes
-   Fix Thermal Tomato Sauce [\#280](https://github.com/EnigmaticaModpacks/Enigmatica9/issues/280) ([MuteTiefling](https://github.com/MuteTiefling))
- Fix missing loot in Industrial Foregoing Quests ([MuteTiefling](https://github.com/MuteTiefling))

File: kubejs/assets/create/lang/en_us.json [hunk 1]
```diff
@@ -15,5 +15,7 @@
 	"block.create.brass_funnel": "Brass Funnel",
 	"block.create.brass_tunnel": "Brass Tunnel",
 	"item.create.brass_hand": "Brass Hand",
-	"item.create.mechanical_arm": "Mehcanical Arm"
+	"item.create.mechanical_arm": "Enchanted Arm",
+	"block.create.mechanical_arm": "Mehcanical Arm",
```
Comment (NielsPilgaard) [inline kubejs/assets/create/lang/en_us.json:19]:
```suggestion
	"block.create.mechanical_arm": "Mechanical Arm",
```

File: kubejs/client_scripts/lang_modifications.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ const entries = {
             }
         },
         {
-            key: 'item.create.mechanical_arm',
+            key: 'block.create.mechanical_arm',
             value: {
                 normal: 'Mehcanical Arm',
```
Comment (NielsPilgaard) [inline kubejs/client_scripts/lang_modifications.js:135]:
```suggestion
                normal: 'Mechanical Arm',
```

File: kubejs/server_scripts/expert/recipes/powah/shaped.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ ServerEvents.recipes((event) => {
     if (global.isExpertMode == false) {
         return;
     }
-    const id_prefix = 'powah:expert/shaped/';
+    const id_prefix = 'enigmatica:base/powah/shaped/';
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/expert/recipes/powah/shaped.js:5]:
```suggestion
    const id_prefix = 'enigmatica:expert/powah/shaped/';
```

**Review by NielsPilgaard — APPROVED** [review]

**Comment by NielsPilgaard** [comment]
Nice work and documentation, as always 👌

**Pull request merged**